rules/cross-platform/execution_aws_ssm_sendcommand_with_command_parameters.toml

```diff
@@ -0,0 +1,103 @@
+[metadata]
+creation_date = "2022/09/03"
+integration = ["endpoint"]
+maturity = "production"
+updated_date = "2024/09/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the use of the AWS Systems Manager (SSM) `SendCommand` API with the either `AWS-RunShellScript` or
+`AWS-RunPowerShellScript` parameters. The `SendCommand` API call allows users to execute commands on EC2 instances using
+the SSM service. Adversaries may use this technique to execute commands on EC2 instances without the need for SSH or RDP
+access. This behavior may indicate an adversary attempting to execute commands on an EC2 instance for malicious
+purposes. This is a [New
+Terms](https://www.elastic.co/guide/en/security/current/rules-ui-create.html#create-new-terms-rule) rule that only flags
+when this behavior is observed for the first time on a host in the last 7 days.
+"""
+false_positives = [
+    """
+    Legitimate use of the `SendCommand` API call to execute commands on EC2 instances using the SSM service may be done
+    by system administrators or DevOps engineers for legitimate purposes.
+    """,
+]
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
```

Comment:
maybe other indexes like auditd ?

Reply:
cc @Aegrah any idea if the query is applicable to auditd or auditd_manager?

Reply:
This is compatible with both Auditd and Auditd Manager, given that you did not add an event.action. It would however be wise to add an event.action, as this will also catch fork events, which you are probably not interested in. 

This would be compatible by adding `event.action in ("exec", "exec_event", "executed", "process_started")` and the corresponding indices : `index = ["logs-endpoint.events.*", "endgame-*", "auditbeat-*", "logs-auditd_manager.auditd-*"]`

example: https://github.com/elastic/detection-rules/blob/main/rules/linux/discovery_dynamic_linker_via_od.toml